tests/test_buffer.py

```diff
@@ -824,6 +824,17 @@ def test_SimpleArray_SimpleArrayPlex_type_switch(self):
         self.assertEqual(
             str(type(arrayplex_int32_2)), "<class '_modmesh.SimpleArray'>")
 
+    def test_sort(self):
```

Comment:
Test `take_along_axis()` too, since you implemented it.

Reply:
Please leave a code comment that there will need another test for ghost indices in the future.